qunicorn_core/__init__.py

```diff
@@ -143,9 +144,18 @@ def create_app(test_config: Optional[Dict[str, Any]] = None):
 
     # To display the errors differently in the swagger ui
     @app.errorhandler(Exception)
-    def handle_internal_server_error(error):
+    def handle_errors(error):
         logging.error(str(error))
-        error_code: int = 500 if not hasattr(error, "status_code") else error.status_code
+        if hasattr(error, "status_code"):
+            error_code = error.status_code
+        elif isinstance(error, ValueError):
+            error_code: int = 404
+        elif isinstance(error, NotImplementedError):
+            error_code: int = 501
+        elif isinstance(error, IBMRuntimeError):
+            error_code: int = 403
+        else:
+            error_code: int = 500
         return {
```

Comment:
Handling the error like this is an easy way and can easily be extended, so error codes of exceptions are understandable, but since all errors of the same type are handled the same way, future implementations that are not aware of this might confuse with the origin of the error code. - if a specific error code like 422 is required for some valueErrors, it can to be added to the HTTP response like below in the deployment_view.py
Is this a valid solution? or should each api endpoint include its own manual error handling?